client/src/router/index.ts

```diff
@@ -9,6 +10,7 @@ const routes = [
     name: 'Landing',
     component: () => import('@/views/landing.vue')
```

Comment:
I think this is for lazy-loading. We should try this in Filings UI for the maintenance filings pages.

Reply:
Good idea, what is method referred to as. Dynamic importing? 😆 Just a wild guess

Reply:
It is dynamic importing, but I think it's called lazy loading. For real.

Reply:
haha awesome :) We should try this! or at least research it a bit more.

Reply:
I just scratched the surface in /bcgov/entity#1890 but now you know something about it too :)